src/codegen/x64/assembler-x64.cc

```diff
@@ -55,7 +55,9 @@ bool OSHasAVXSupport() {
   // All replaying happens on linux so skip macOS checks in that case which
```

Comment:
Comment needs fixing?

Reply:
No, the comment is right, we only replay on linux.  The code below is testing for some processor features related to the jitcode being generated, and while it's a little mysterious to me why the macOS version is relevant to that, there isn't a need to do this checking when we're replaying on linux.